cogs/stars.py

```diff
@@ -1,30 +1,33 @@
-import aiohttp
+from aiohttp import ClientSession
+from nextcord import TextChannel
 from nextcord.ext import commands
 from nextcord.ext.tasks import loop
 
 
+STARS_CHANNEL_ID = 884441198520045570
+
 class GitHubStars(commands.Cog):
     """Run a task loop to update a channel with the current stars for nextcord/nextcord and nextcord/nextcord-v3"""
 
-    STARS_CHANNEL_ID = 884441198520045570
-
     def __init__(self, bot: commands.Bot):
-        self.__bot = bot
+        self.bot = bot
         self.update_stars.start()
 
+
     async def get_stars(self, repo: str) -> int:
         """Get number of GitHub stars for a given repo in the form 'owner/repository'"""
-        async with aiohttp.ClientSession() as session:
+        async with ClientSession() as session:
```

Comment:
I don't see the point of changing the way this is imported. Both ways work

Reply:
I expect it to reduce 0.0000000001 second or something to optimize, as we don't need anything else from the library- hence importing just what we need would be better.

Reply:
Importing the module rather than the class doesn't waste anything; the module is always fully imported. The only difference is the name you write when using the stuff.

Also, even if it did take 0.00000001 seconds longer, do you think anyone would be able to notice the difference when starting up the bot?

Reply:
oh.
also oh.